docs/getting-started.md

```diff
@@ -17,18 +17,31 @@ or if you have composer installed globally
 composer require robthree/twofactorauth
 ```
 
+**Note:** If you are not using a composer-aware framework, you should [include the composer loader yourself](https://getcomposer.org/doc/01-basic-usage.md#autoloading).
+
```

Comment:
I wouldn't have bothered with this, if TwoFactorAuth is the first time someone has realised they need to include the composer autoloader, they probably have other issues.

Reply:
Yeah, I thought about: `If you are not using composer, what the hell are you doing??`, but removing it entirely seems best ;)

Same with the two different ways to use composer. Most libs just indicate the `composer require` not the phar one, and users should be smart enough to figure out which one they need. So I'll simplify that, too.

Reply:
Oh yes good spot